Automattic/newspack-plugin
# Visually update the Welcome step

### All Submissions:






### Changes proposed in this Pull Request:

In order to slightly update the design of the Welcome step, this PR does a few things:

* Adds a new `isMedium` prop to the `Card`
* Reworks `NewspackLogo` to accept `height`, `className`, and `centered` props
* Adds css for H1
* Reorganises the Welcome screen to use new stuff listed above and adds textdomain to strings

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test the onboarding
3. Check the other wizards if the changes broke anything

### Other information:
